### PR TITLE
Created: works section & about me

### PR DESCRIPTION
**Pull request by Amrhub**
Added markup for works section and about me in the portfolio for mobile version 
Added CSS for HTML works section and about me Some images were added for CSS purposes
Updated README.md file to match these changes

### CODE REVIEW

**Comment by Amrhub** [comment]
I  just saw that Linter failed so I will check it I tried to undo-submission but it won't let me so please hold on don't review it

**Review by EricMbouwe — CHANGES_REQUESTED** [review]
Hi @Amrhub, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has linters falling to the checks. You should make sure that your repo looks professional before review.
Please take a step back and read a lesson [here](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Amrhub** [comment]
I am sorry It's my first time fixing linter errors so committed multiple times :sweat:

File: index.html [hunk 1]
```diff
@@ -22,10 +22,151 @@ <h1 class="myName">
                     Amr Ahmed
                 </h1>
                 <h3 class="role">Software Developer</h3>
-                <p class="self-intro">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,  don’t hesitate to contact me.</p>
+                <p class="self-intro">I can help you build a product , feature or website Look tdrough some of my work and experience! If you like what you see and have a project you need coded,  don’t hesitate to contact me.</p>
             </section>
+
```
Comment (icebox827) [inline index.html:27]: - [ ] **REQUIRED:** To write better and clean code, have your semantic HTML element (header, section, and so on...) outside of a div tag, not the opposite. For example:
```
<header>
    <div></div>
<header>

<section>
   <div></div>
<section>

File: index.html [hunk 2]
```diff
@@ -22,10 +22,151 @@ <h1 class="myName">
...
+                        <li>Codepen</li>
+                        <li>Gitlab</li>
+                        <li>Terminal</li>
+                    </ul>
+                </section>
+                
+            </div>
+            <div class="links">
+                <a href="#">
+                    <img src="/images/Icons/GitHub-icon.png" alt="GitHub icon" >
```
Comment (icebox827) [inline index.html:68]: - [ ] **REQUIRED:** Please wrap all the links in an ul and li tag

File: css/style.css [hunk 3]
```diff
@@ -1,36 +1,43 @@
...
+
 .top-container {
   display: flex;
   flex-direction: column;
   padding: 20px 40px;
   background-image: url('/images/headline_section_bg.png');
   background-repeat: no-repeat;
   background-size: cover;
-  height: 100vh;
+  background-position: center;
```
Comment (icebox827) [inline css/style.css:35]: - [ ] **REQUIRED:** It will be nice to have the main container of the projects has a display:grid

**Review by icebox827 — CHANGES_REQUESTED** [review]
Hi Ahmed, 

Good job so far!
Your project looks good, but there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/style.css (hunk 3 above)
Comment (Amrhub) [reply css/style.css:35]: But .top-container is not the main container so in this case, I will make .wrapper display: grid

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @Amrhub , 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**